### ISSUE TITLE
WCAG2ICT Project: Add from WCAG 2.2 - 2.5.8 Target Size (Minimum)

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#40]
- [x] Add [2.5.8 Target Size (Minimum)](https://www.w3.org/TR/WCAG22/#target-size-minimum) section (from WCAG 2.2)
- [x] Determine if new definitions, "[pointer input](https://www.w3.org/TR/WCAG22/#dfn-pointer-inputs)", "[target](https://www.w3.org/TR/WCAG22/#dfn-targets)" and "[target offset](https://www.w3.org/TR/WCAG22/#dfn-target-offsets)" used in this SC need interpretation. Per Issue #22. As of 21 Feb. 2023, none of these definitions have any web-centric language so should be able to apply as written.
- [x] Add in the [Understanding 2.5.8 Target Size (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) Intent and Benefits sections
- [x] Consider and write up whether there are any notes needed for application of this SC to non-web technologies

**Comment by pday1** [comment]
Proposed content in issue #80

**Comment by pday1** [comment]
Revised content to be discussed in issue https://github.com/w3c/wcag2ict/issues/80

**Comment by maryjom** [comment]
The TF reached [consensus on the guidance for SC 2.5.8](https://www.w3.org/2023/06/08-wcag2ict-minutes.html#t03) on 8 June, and reached consensus on an [update to a non-web document note](https://github.com/w3c/wcag2ict/wiki/WCAG2ICT-Task-Force-Decisions) on 29 June. The AG WG [reviewed 4.1.3](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-Third-content-review-by-AGWG2/results#xq14) in the 3rd content review and approved the content on 25 July.
